.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=lf
```

Comment:
Not an expert on this (just read the docs). Could we have:

    * text=auto
    *.sh text
    *.scala text
    [...]
    *.jar binary
    *.png binary

Etc.? I'd rather have an explicit list of extensions than open ourselves to surprises (e.g. when one of us upgrades Git).